modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,88 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install logging add-on service on {product-title}
+
+{product-tite} (ROSA) provides logging through the `cluster-logging-operator` add-on. This add-on service offers an optional application log forwarding solution based on AWS CloudWatch. This logging solution can be installed after the ROSA cluster is provisioned.
+
+.Procedure
+
+. Enter the following command:
++
+[source,terminal]
+----
+$ rosa install addon cluster-logging-operator --cluster=<cluster_name> --interactive
+----
++
+For `<cluster_name>`, enter the name of your cluster.
+
+. When prompted, accept the default `yes` to install the `cluster-logging-operator`.
+. When prompted, accept the default `yes` to install the optional Amazon CloudWatch log forwarding add-on or enter `no` to decline the installation of this add-on.
```

Comment:
@jonquilwilliams FYI, currently the default value is not 'yes' for all the parameters, its 'no'. 
There is a bug created for this: https://issues.redhat.com/browse/LOG-1060

Reply:
@gkarager The code is fixed and in code review under https://issues.redhat.com/browse/SDA-3676

So the documentation should be correct as written for GA.